1.raspberry-pi-stuff/octoberry/octoberry/__main__.py

```diff
@@ -24,17 +29,29 @@ def play_sound(sound_url, cached_sound_path: None | str=None) -> None:
         cached_sound_path = os.path.join(os.getcwd(), os.path.basename(parsed_url.path))
 
     if not os.path.exists(cached_sound_path):
-        print("Downloading sound...")
+        logger.info("Downloading sound...")
         r = requests.get(sound_url, allow_redirects=True)
         open(cached_sound_path, 'wb').write(r.content)
-        print("Sound downloaded")
+        logger.info("Sound downloaded")
+
+    logger.info("Playing sound...")
+    subprocess.call(f'/usr/bin/mpv {cached_sound_path}', shell=True)
+
+
+def play_sounds(sound_urls: list[str]) -> None:
+    '''
+    Play list of sounds from urls
+
+    Args:
+        sound_urls (list[str]): list of urls to sound files
 
-    print("Playing sound...")
-    mixer = mixer.init()
-    mixer.music.load(cached_sound_path)
-    mixer.music.play()
+    Returns:
+        None
+    '''
+    for sound_url in sound_urls:
+        _play_sound(sound_url)
 
-def read_rfid() -> tuple[str, str] | None:
+def read_rfid() -> Union[Tuple[str, str], None]:
     '''
     Read RFID from RFID reader
     We use EH301 RFID reader which simulate keyboard input (with newline)
```

Comment:
the steps to configure the device are not correct, it was before we decided to use the default config.

I would still doc the default config because nothing guarantees that another device will come from the factory with the same configs